docker-app/qfieldcloud/core/exceptions.py

```diff
@@ -175,3 +175,23 @@ class ReachedMaxOrganizationMembersError(QFieldCloudException):
     code = "organization_has_max_number_of_members"
     message = "Cannot add new organization members, account limit has been reached."
     status_code = status.HTTP_403_FORBIDDEN
+
+
+class AccountInactiveError(QFieldCloudException):
+    """
+    Raised when a permission (action) is denied because the useraccount is inactive.
+    """
+
+    code = "permission_denied_inactive"
+    message = "Permission denied because the useraccount is inactive"
+    status_code = status.HTTP_403_FORBIDDEN
+
```

Comment:
I reverted back to 403_FORBIDDEN instead of 402_PAYMENT_REQUIRED because IMO for an inactive community there is no payment required?
Please correct if this payment required is indeed the only possible reason for inactive status?

Reply:
Let's merge it liike this and later we can change.